docs/website/docs/reference/installation.md

```diff
@@ -109,3 +109,17 @@ You can install `dlt` in your virtual environment by running:
 ```sh
 pip install -U dlt
 ```
+
+## Install dlt via pixi and conda
```

Comment:
now we have different style for dlt (((
![image](https://github.com/dlt-hub/dlt/assets/20367975/0ae21257-3744-4448-a142-1e43f9303c7a)

Reply:
Did you mean to move dlt inside `?